pkg/controller/deployment/deployment_controller.go

```diff
@@ -120,7 +154,12 @@ func (d *DeploymentController) reconcileRollingUpdateDeployment(deployment exten
 		// Update DeploymentStatus
 		return d.updateDeploymentStatus(allRCs, newRC, deployment)
 	}
-	// TODO: raise an event, neither scaled up nor down.
+
+	// raise an event, neither scaled up nor down.
+	if !scaledUp && !scaledDown {
+		d.eventRecorder.Eventf(&deployment, api.EventTypeNormal, "ScalingRC", "Scaling not required")
```

Comment:
Doesn't this mean that every time a full reconciliation occurs (during startup, or during the reflector resync period) we'll generate "Nothing happened" events for all deployments using the recreate strategy? If so, I'd suggest we remove the event.